Make the `documentation' target stop depending on basically all other targets,
since we are only interested in building the cmake-gui one. If we don't do
this, building the documentation will build all other targets which are only
needed by devel/cmake itself.
--- Utilities/Sphinx/CMakeLists.txt
+++ Utilities/Sphinx/CMakeLists.txt
@@ -93,19 +93,6 @@ endforeach()
 
 add_custom_target(documentation ALL DEPENDS ${doc_format_outputs})
 
-foreach(t
-    cmake
-    ccmake
-    cmake-gui
-    cpack
-    ctest
-    )
-  if(TARGET ${t})
-    # Build documentation after main executables.
-    add_dependencies(documentation ${t})
-  endif()
-endforeach()
-
 if(SPHINX_MAN)
   file(GLOB man_rst RELATIVE ${CMake_SOURCE_DIR}/Help/manual
     ${CMake_SOURCE_DIR}/Help/manual/*.[1-9].rst)
